src/components/DataViewFilterDropdownButtons/DataViewFilterDropdownButtons.styled.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+import Button from "../Button";
+import ButtonRow from "../ButtonRow";
+
+export const StyledWrapper = styled.div`
+  & {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 1rem;
+  }
+`;
+
+export const StyledButton = styled(Button)`
+  &.normalButton > button {
+    font-size: 16px;
+  }
+`;
+
+export const StyledButtonRow = styled(ButtonRow)`
+  .button + .normalButton {
+    margin: 0;
+  }
+
+  .button:first-child {
+    border-right: 2px solid ${theme.colors.gray200};
+    padding-right: 16px;
```

Comment:
Buttons already have 8px padding, so we should only add 8px extra to sum a total of 16. (same with the padding left of the right child).